src/pages/LandingPage.py

```diff
@@ -31,4 +35,10 @@ def get_upcoming_public_event(self):
         self.scroll_down_page()
         return self.find_element_by_css(self.UPCOMING_PUBLIC_EVENT)
 
+    def click_join_event_link(self):
+        self.scroll_down_page()
+        time.sleep(2)
+        return self.join_event_link.click_link_by_xpath()
```

Comment:
As I understand you scroll down to the link so that it will appear but as I see, the link is present in the code without scrolling down there.

Reply:
Fixed in new commit